.github/workflows/release-keyring.yaml

```diff
@@ -0,0 +1,37 @@
+name: Artifacts Helper Keyring Releases
+
+on:
+  push:
+    tags:
+      - "feature_artifacts-helper_*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: src/artifacts-helper/codespaces_artifacts_helper_keyring
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: wntrblm/nox@main
+        with:
+          python-versions: "3.11, 3.12"
+
+      - name: Setup PDM
+        uses: pdm-project/setup-pdm@v4
+        with:
+          version: "2.15.1"
+          python-version-file: "src/artifacts-helper/codespaces_artifacts_helper_keyring/pyproject.toml"
+
+      - name: Run nox release
+        run: pdm release
+
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: codespaces_artifacts_helper_keyring
+          path: |
+            src/artifacts-helper/codespaces_artifacts_helper_keyring/dist
```

Comment:
@markphip, this only uploads the built package as a build artifact on the workflow. I've also got this snippet if you wanted to automatically create a release and attach the artifact.
```yaml
      # Additionally create a release and upload the artifacts
      - uses: ncipollo/release-action@v1
        with:
          artifacts: "src/artifacts-helper/codespaces_artifacts_helper_keyring/dist/codespaces_artifacts_helper_keyring-*"
          artifactErrorsFailBuild: true
          body: |
            This is a release of the Codespaces Artifacts Helper Keyring.
            You should not have to manually install this package; it will be installed
            automatically by the Codespaces Artifacts Helper feature.
```

Reply:
For now I want to do it manually so lets leave it attaching artifact to the workflow run. Please change the workflow so that it can also be triggered on workflow_dispatch and take your PR out of Draft when you feel it is ready.

Reply:
Done!